### ISSUE TITLE
TextField::initRenderer() missing font

### ISSUE THREAD

**Issue by MrCapone** [cocos2d/cocos2d-x#18403]
When i trying to create `TextField` by 
```
TextField::create("some_text", "some_font_from_resources", 18);
```
I got log warning
> cocos2d: fullPathForFilename: No file found at Thonburi. Possible missing file

breakpoint brings me to this [line](https://github.com/cocos2d/cocos2d-x/blob/v3/cocos/ui/UITextField.cpp#L335). What is **Thonburi** and why it missing?

**Comment by drelaptop** [comment]
__Thonburi__ is a font name, you can find `Thonburi.ttf` in the engine. but I think it's not a good method to init `TextField ` with a determinate font, I will try to find a method to solve it.

**Comment by drelaptop** [comment]
this Issue has been fixed, pleased see PR #18518
